vtex-apps/geolocation-graphql-interface
# CHK-316: Initial Setup

#### Description

The setup is based on [this document](https://docs.google.com/document/d/11DewV9j4qiz_U_tuLsAHUa235oydbcSupIkuVdMi9kg), https://github.com/vtex-apps/graphql-interface-app-example and https://github.com/vtex-apps/search-graphql (which is also an interface).

The schema is based on a subset of https://github.com/vtex-apps/places-graphql/ schemas (see https://github.com/vtex-apps/places-graphql/pull/5). After CHK-340 is complete, this subset of schemas on Places GraphQL will be removed.

#### How should this be manually tested?

Just check if the files are correct and make sense, there is not too much to test besides that. I suggest reviewing per commit.
